src/empire/object.c

```diff
@@ -168,14 +264,19 @@ int empire_object_init_distant_battle_travel_months(int object_type)
     return month;
 }
 
-const empire_object *empire_object_get(int object_id)
+full_empire_object *empire_object_get_full(int object_id)
```

Comment:
Can't  this function and the next return a `const` instead?

Reply:
No, they get modified in empire_xml